tests/testthat/test-utils.R

```diff
@@ -0,0 +1,8 @@
+test_that("generic utils work", {
+  ve_c <- version_cairo()
+
+  expect_s3_class(ve_c, "package_version")
+  expect_s3_class(ve_c, "numeric_version")
+
+  expect_true(match_family("serif") %in% c("Times New Roman", "DejaVu Serif"))
```

Comment:
This will not work on CRAN platform. We don't know what font they have. I would remove that ; Deja Vu can be missing from Linux, not sure but I think TNR is installed with Office, so it won't be there on CRAN